h/services/bulk_annotation.py

```diff
@@ -58,19 +54,19 @@ def date_match(column: sa.Column, spec: dict):
     return sa.and_(*clauses)
 
 
+@dataclass
+class BulkAnnotation:
+    username: str
+    authority_provided_id: str
+
+
 class BulkAnnotationService:
     """A service for retrieving annotations in bulk."""
 
     # Aliases to distinguish between different types of user
     _AUTHOR = sa.orm.aliased(User, name="author")
     _AUDIENCE = sa.orm.aliased(User, name="audience")
 
-    # Acceptable values to pass in as `fields`
-    _FIELDS = {
-        "author.username": _AUTHOR.username,
-        "group.authority_provided_id": Group.authority_provided_id,
-    }
```

Comment:
So the only acceptable fields were `"author.username"` and `"group.authority_provided_id"`